test/controllers/favorite_recipes_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class FavoriteRecipesControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
```

Comment:
@Za-hell You created routes for new, create, index,edit and update. I think we also need a destroy as people might not want the recipe in their favourites anymore.
Also need the actions in here. Thanks!
S